pre_commit/languages/docker.py

```diff
@@ -76,7 +78,24 @@ def install_environment(
         os.mkdir(directory)
 
 
+@functools.lru_cache(maxsize=1)
+def docker_is_rootless() -> bool:
+    returncode, stdout, stderr = cmd_output(
+        ('docker', 'system', 'info'),
+    )
+    for line in stdout.splitlines():
+        # rootless docker has "rootless"
+        # rootless podman has "rootless: true"
+        if line.strip().startswith('rootless'):
+            if 'false' not in line:
+                return True
+            break
+    return False
+
+
 def get_docker_user() -> Tuple[str, ...]:  # pragma: win32 no cover
+    if: docker_is_rootless():
```

Comment:
This small typo makes the CI explode:
```suggestion
    if docker_is_rootless():
```
(just accepting it via GH UI should fix that)

Reply:
done